### DISCUSSION TITLE
튜토리얼 관련 질문

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/block-it#37] [General]
[튜토리얼 issue](https://github.com/junnie082/block-it/issues/10)
1. 왜 2번과 3번창이 두개 인가요?
2. 아래 그림과 같이 하나의 그림과 간결한 설명이 모바일 환경 상 더욱 알맞는 튜토리얼 같습니다. 다른 창도 이렇게 만들어주세요. 설명은 그림 상단과 하단을 나누었기에 필요 시 설명 앞에 "[상단]"과 같이 넣어주세요.
![Image](https://user-images.githubusercontent.com/88719152/234158946-44906e09-b618-46a0-a128-68ae2778c3c3.png)

**Comment by gomdoricake** [comment]
1. 2번 창은 무엇이고 3번 창은 무엇인가요?
2. "설명은 그림 상단과 하단을 나누었기에 필요 시 설명 앞에 "[상단]"과 같이 넣어 주세요." 에 대한 설명이 더 필요합니다.

**Reply by seong0929** [reply]
1. 튜토리얼 페이지 2페이지와 3페이지가 두개에요
2. 상단 설명, 그림, 하단 설명 구조로 되어 있으니, 추후 그림과 설명을 적어줄 때 어디에 쓸 자료인지 알려주세요

**Reply by gomdoricake** [reply]
1. 2번 페이지에서 "Press this button" 이라고 적힌 노란색 원을 누르면 말이 움직입니다. 3번 페이지에서 "Put" 을 누르면 플랭크가 보드판에 놓입니다. 피그마의 삼각형 버튼을 눌러 애니메이션으로 이를 확인하실 수 있습니다.
2. 한 페이지 당 [상단], [그림], [하단] 구조로 나뉘어 있으니, 이를 더 명확하게 하기 위해 각 부분마다 "상단", "그림", "하단" 글씨를 써 넣으면 되는 것인가요?

**Reply by seong0929** [reply]
1. 피그마 형식을 그대로 하나요? 아니면 페이지를 넘겨 다음 페이지로 할까요? 2번 작업을 하면서 수정해서 전달해주세요
2. 네
